drud/ddev
# Support go modules for build

## The Problem/Issue/Bug:

Go modules allows the ddev project to be built in any directory (not just in GOPATH), and with a directory of any name, because the module name is in the go.mod file. 

* go mod init github.com/drud/ddev (creates go.mod from Gopkg.toml
* go mod tidy (gets all the deps in the test files and such)
* This does *not* do `go mod vendor`, which would put go mod in charge of vendor directory.
* It seems to be completely compatible with current build-tools and golang-build-container.
* As we get comfortable with this we'll likely want to remove the vendor directory.

## How this PR Solves The Problem:

## Manual Testing Instructions:

* You should be able to build and test in any directory (In other words, it should build and test in ~/workspace/ddev_something
